src/loaders/compute_tools/checkm2/checkm2.py

```diff
@@ -68,6 +69,13 @@ def _run_checkm2(
         fatal_tuples.append(fatal_tuple)
     write_fatal_tuples_to_dict(fatal_tuples, output_dir)
 
+    metadata = {'tool': 'checkm2',
+                'version': '1.0.1',
+                'command': command,
+                'reference_db': 'diamond_db ver unknown - saved copy in NERSC at global/cfs/cdirs/kbase/collections/libraries/checkm2/1.0.1/uniref100.KO.1.dmnd',
```

Comment:
```suggestion
                 "reference_db": {
                     "path": "/global/cfs/cdirs/kbase/collections/libraries/checkm2/1.0.1/uniref100.KO.1.dmnd",
                     "location": "NERSC",
                     "version": None,
                     "comment": "diamond_db, ver unknown",
                 },
```

Reply:
Although it seems like the path has the same issue as the version in that it could change and not be updated here. Also I suppose the container could be run at a different facility with a different path so maybe those 2 fields are ill advised

Reply:
I think that's probably fine at the moment.

Reply:
If we're talking about using Lawrencium in the future I don't think we should add data that we know isn't portable

Reply:
👍 removed location and path.

Reply:
:+1: